src/main/java/taxi/model/Driver.java

```diff
@@ -4,15 +4,19 @@
 
 public class Driver {
```

Comment:
It's better to add `toString()` method here, to get more informative exception messages in dao layer (but do not include password field there)